bin/g5-transmitter.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+function usage
+{
+  echo "Usage:"
+  echo "cgm-transmitter [transmitter_id]"
+  echo
+  exit
+}
+
+function  generate_config
+{
+  echo "{\"transmitter_id\":\"${txId}\",\"sensor_code\":\"\",\"mode\":\"not-expired\",\"pump_units\":\"mg/dl\",\"fake_meter_id\":\"000000\"}" | jq . > $config
+}
+
+numArgs=$#
+if [ $(bc <<< "$numArgs < 1") -eq 1 ] || [ $(bc <<< "$numArgs > 1") -eq 1 ]; then
+  usage
+  exit
+fi
+
+txId=$1
+
+#validate transmitter id
```

Comment:
Nice - this will prevent a lot of typo errors and avoid some troubleshooting for sure.